deployment/terraform/templates/cloud-config.tpl

```diff
@@ -15,4 +22,6 @@ runcmd:
   - mkdir -p /etc/docker/certs
   - unzip -d /etc/docker/certs /tmp/certs.zip
   - echo 'OPTIONS="$${OPTIONS} -H unix:///var/run/docker.sock  -H 0.0.0.0:2476 --default-ulimit nofile=1024:4096 --tlsverify --tlscacert=/etc/docker/certs/ca.pem --tlscert=/etc/docker/certs/server.pem --tlskey=/etc/docker/certs/server.key"' >> /etc/sysconfig/docker
-  - service docker restart
\ No newline at end of file
+  - service docker restart
+  - stop ecs
+  - sed -i 's/start on/#start on/' /etc/init/ecs.conf 
```

Comment:
I think that it would be better to do this via:

```bash
$ echo manual | sudo tee /etc/init/ecs.override
```

That blocks automatic Upstart service spinup and will work across reboots. You may also want to add that under `bootcmd` vs. `runcmd` so that it gets drops down earlier in the boot sequence.